### PR TITLE
Rewrite`Dockerfile`

### PR DESCRIPTION
**Pull request by smondet**
One could use in every call in the meantime:

     Program.(sh "apt-get install --yes wget" && p)

Fixing #336.

---
This change is [Reviewable](https://reviewable.io/reviews/hammerlab/biokepi/337)

### CODE REVIEW

File: tools/docker/run/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,18 @@
-FROM ubuntu:16.04
-RUN apt-get update
-RUN apt-get install --yes cmake r-base tcsh libx11-dev libfreetype6-dev pkg-config
+# We pull form `ketrew-server` beause the user management has been done right
```
Comment (ihodes) [inline tools/docker/run/Dockerfile]: *from

What do you mean by "user management has been done right"?

Comment (smondet) [inline tools/docker/run/Dockerfile]: See there: https://github.com/ocaml/opam-dockerfiles/blob/ubuntu-16.04_ocaml-4.03.0/Dockerfile#L10

**Comment by smondet** [comment]
It built itself successfully: https://hub.docker.com/r/hammerlab/biokepi-run/builds/bfmwi7xgcjf2jmiafpsomdz/

**Comment by smondet** [comment]
And with that container I can report, a successful use of `java`:

```
[Wed Aug 17 18:31:13 UTC 2016] picard.sam.CreateSequenceDictionary REFERENCE=/nfsresults/coclobas-ketrew-biokepi-test/refgenomes/b37/b37.fasta OUTPUT=/nfsresults/coclobas-ketrew-biokepi-test/refgenomes/b37/b37.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false
[Wed Aug 17 18:31:13 UTC 2016] Executing as opam@64875d1c-e07c-5be3-81ab-020580714a37 on Linux 3.16.0-4-amd64 amd64; Java HotSpot(TM) 64-Bit Server VM 1.7.0_80-b15; Picard version: 1.127(ec9d735ca1aef6e7bea2f1f84e515acb1f7cbe32_1418660495) IntelDeflater
[Wed Aug 17 18:32:57 UTC 2016] picard.sam.CreateSequenceDictionary done. Elapsed time: 1.73 minutes.
Runtime.totalMemory()=840957952
```

**Comment by smondet** [comment]
I ran most of epidisco on dream-chr20 (with tumor as fake RNA) with this version!

pipeline

**Pull request merged**